components/tools/OmeroPy/src/omero/plugins/fs.py

```diff
@@ -272,6 +276,19 @@ def _configure(self, parser):
             "fileset",
             type=ProxyStringType("Fileset"))
 
+        logfile = parser.add(sub, self.logfile)
+        logfile.add_argument("fileset", type=ProxyStringType("Fileset"))
+        logfile.add_argument(
+            "filename",  nargs="?", default="-",
+            help="Local filename to be saved to. '-' for stdout")
```

Comment:
Not sure our initial capitalization is consistent but CLI help is confused enough I've no idea which is favored, e.g.,
```
  -h, --help                            show this help message and exit
  --javahelp, --java-help               Show the Java help text
```
so no strong opinions for this PR.